components/FAQSection/FAQSection.test.tsx

```diff
@@ -0,0 +1,9 @@
+import { screen, render } from "@testing-library/react";
```

Comment:
Let's add a tests so we make use this variable `screen`. An simple idea could be:

- Check if there is a `heading` of _level 2_ present within the section